src/zendesk_mcp_server/zendesk_client.py

```diff
@@ -10,10 +10,11 @@
 
 
 class ZendeskClient:
-    def __init__(self, subdomain: str, email: str, token: str):
+    def __init__(self, subdomain: str, email: str, token: str, locale: str = "en-us"):
```

Comment:
No need to set the default twice.

```suggestion
    def __init__(self, subdomain: str, email: str, token: str, locale: str):
```